source/expressions.tex

```diff
@@ -5953,18 +5953,24 @@
 \end{note}
 
 \pnum
```

Comment:
Maybe drop the paragraph number too, as we're continuing the same case?

Reply:
fixed